### PR TITLE
Update signals.rst

### PR DESCRIPTION
**Pull request by jacques-oliveira**
Updating Timer function for version 4.x

### CODE REVIEW

File: getting_started/step_by_step/signals.rst [hunk 1]
```diff
@@ -300,6 +300,8 @@ We can now connect the Timer to the Sprite2D in the ``_ready()`` function.
     func _ready():
         var timer = get_node("Timer")
         timer.timeout.connect(_on_timer_timeout)
+        #for version 4.x use
+        #timer.connect("timeout", _on_timer_timeout)
```
Comment (AThousandShips) [inline getting_started/step_by_step/signals.rst:304]: This is incorrect, the correct syntax is the above, and this *is* for 4.x, so this change doesn't make sense

**Review by AThousandShips — CHANGES_REQUESTED** [review]

File: getting_started/step_by_step/signals.rst (hunk 1 above)
Comment (jacques-oliveira) [reply getting_started/step_by_step/signals.rst:304]: Hi @AThousandShips , to me the line 302 doesn't works, an alternative code I used was the line 304 to godot version 4.1.3,  be free to change anything you want, thank you.

Comment (AThousandShips) [reply getting_started/step_by_step/signals.rst:304]: Tested and yes the code works, so likely an error in copying the code

This should be closed

Comment (AThousandShips) [reply getting_started/step_by_step/signals.rst:304]: Have you confirmed you did not make this mistake?

**Comment by mhilbrunner** [comment]
Closing as per the above discussion; this code is correct for 4.0+.

**Pull request closed without merging**